lib/jekyll-resize.rb

```diff
@@ -7,17 +7,20 @@ module Resize
     HASH_LENGTH = 32
 
     # Generate output image filename.
-    def _dest_filename(src_path, dest_dir, resize_option, imageFormat)
+    def _dest_filename(src_path, dest_dir, options, imageFormat)
       base_name = File.basename(src_path, File.extname(src_path))
-      options_slug = resize_option.gsub(/[^\da-z]+/i, "")
       ext = imageFormat && !imageFormat.empty? ? ".#{imageFormat}" : File.extname(src_path)
 
-      "#{base_name}_#{options_slug}#{ext}"
+      example_string = "#{base_name}_#{options}#{ext}"
+      "#{example_string.hash}#{ext}"
```

Comment:
This seems to implicitly write the hash as decimal - did we intend to explicitly return hex?